services/messageService.js

```diff
@@ -0,0 +1,51 @@
+import { AsyncStorage } from 'react-native';
```

Comment:
Love the refactor here, servicification is almost always better down the road. Now that the view is decoupled from the logic, we could potentially start adding tests here.

Reply:
Yeah tests is the next big priority item. The ultimate goal is to migrate this messaging logic into redux, so I need to figure out how long we expect to be in this state before going to redux.